reporter/src/main/resources/template/asciidoc.ftl

```diff
@@ -0,0 +1,152 @@
+[#--
+Copyright (C) 2020 HERE Europe B.V.
```

Comment:
This should also get our Copyright, or maybe only our Copyright, depending on how much / if code was copied from the existing templates.

Reply:
I would say both, since I first copied everything from `default.ftl` and then reduced and modified it. Would you agree?

Reply:
Yes.